python/cproton.py

```diff
@@ -543,7 +543,7 @@ def pn_receiver(session, name):
 
 
 def pn_delivery(link, tag):
-    return lib.pn_delivery(link, py2bytes(tag))
+    return lib.pn_delivery(link, py2bytes(tag, 'latin-1'))
```

Comment:
So this change makes sense from the point of view of round tripping the tags. But it is a change of behavior which will raise an exception for some input strings which would previously have been accepted (but with a different encoding).